lib/assets/i18n/log/messages_en.json

```diff
@@ -106,6 +94,7 @@
   "ZWED0123I":"%s",
   "ZWED0124I":"Plugin %s at path=%s loaded.",
   "ZWED0125I":"SessionStore.sessions.size=%d",
+  "ZWED0126I":"SessionStore.sessions.size=%d",
```

Comment:
Duplicate?

Reply:
JP wanted to explicitly have duplicates to mark where in the code something occurred. We happened to have some messages that appeared twice.